### PR TITLE
Add opset 16 support and check ci

### PR DESCRIPTION
**Pull request by hwangdeyu**
https://github.com/onnx/tensorflow-onnx/issues/1935
Signed-off-by: Deyu Huang <deyhuang@microsoft.com>

### CODE REVIEW

File: ci_build/azure_pipelines/templates/job_generator.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ parameters:
   python_versions: ['3.7']
   tf_versions: ['']
   onnx_versions: ['']
-  onnx_opsets: ['15', '14', '13', '12', '11', '10', '9']
-  onnx_backends: {onnxruntime: ['1.10.0']}
+  onnx_opsets: ['16', '15', '14', '13', '12', '11', '10', '9']
+  onnx_backends: {onnxruntime: ['1.11.0']}
```
Comment (hwangdeyu) [inline ci_build/azure_pipelines/templates/job_generator.yml:9]: ORT 1.11 is not supported Python 3.6.

File: ci_build/azure_pipelines/unit_test.yml [hunk 2]
```diff
@@ -128,7 +128,7 @@ stages:
     - template: 'templates/job_generator.yml'
       parameters:
         # tf 1.12
-        python_versions: [3.6']
+        python_versions: [3.7']
         tf_versions: ['1.12.3']
```
Comment (hwangdeyu) [inline ci_build/azure_pipelines/unit_test.yml:132]: `opset16` requires -> `ort1.11` -> `python >= 3.7` -> `tf >= 1.13.1`

File: ci_build/azure_pipelines/templates/setup.yml [hunk 3]
```diff
@@ -4,14 +4,22 @@ steps:
 - bash: |
     set -ex
     pip install pytest pytest-cov pytest-runner coverage graphviz requests pyyaml pillow pandas parameterized
-    pip install $(CI_PIP_TF_NAME) $(CI_PIP_ONNX_NAME) $(CI_PIP_ONNX_BACKEND_NAME)
+    pip install $(CI_PIP_TF_NAME) $(CI_PIP_ONNX_NAME)
 
+    # TF < 2.7 reuires numpy <= 1.19, but onnxruntime >= 1.11 requires numpy >= 1.21
+    if [[ $CI_TF_VERSION < 2.7 ]] && [[ $CI_ONNX_BACKEND == "onnxruntime" ]] ;
+    then 
+        pip install $(CI_PIP_ONNX_BACKEND_NAME) numpy --no-deps -U
```
Comment (fatcat-z) [inline ci_build/azure_pipelines/templates/setup.yml:12]: Should we specify the version of numpy to 1.19.0 here for safety?

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

**Pull request merged**